tools/ci-build/changelogger/src/render.rs

```diff
@@ -97,18 +101,76 @@ pub fn subcommand_render(args: &RenderArgs) -> Result<()> {
             .unwrap_or(current_dir.as_path()),
     )
     .context("failed to find smithy-rs repo root")?;
-    let smithy_rs = GitCLI::new(&repo_root)?;
 
+    let current_tag = {
+        let cli_for_tag = if let Some(aws_sdk_rust_repo_root) = &args.aws_sdk_rust_location {
```

Comment:
Presumably a prerequisite of running this is that the `aws-sdk-rust` repo is on `main` and that the local is up to date with `origin/main`? If there a way we can enforce that and fail if it isn't true?

Reply:
No there isn't, but only the release pipeline renders changelog for `aws-sdk-rust` in the release pipeline. It clones the `aws-sdk-rust` repo and checks out from `origin/main`. It's been like that over a couple of years now. So I'd keep what we have here.